client/src/components/editor/Icon.js

```diff
@@ -24,9 +24,9 @@ const Icon = ({ icon, title, className }) => {
 
   return (
     <svg
-      width="16"
-      height="16"
-      viewBox="0 0 1024 1024"
+      width={width || "16"}
+      height={height || "16"}
+      viewBox={viewBox || "0 0 1024 1024"}
```

Comment:
Make these props required and set a default.

Reply:
I agree about setting a default instead of the current implementation, but why make them required?

Reply:
You're right, add a default (then they won't be required).